packages/synapse-react-client/src/style/components/entity_finder/_entity-finder.scss

```diff
@@ -57,15 +56,9 @@ $-splitter-width: 12px;
     height: $-finder-height + ($-border-size * 2);
   }
 
-  &__MainPanel.MainPanelSearch {
-    width: 85vw;
```

Comment:
These `vw` units caused overflow when 85% of the viewport width was greater than the maximum MUI dialog content width